packages/core/src/interpreter.ts

```diff
@@ -464,13 +464,11 @@ export class Interpreter<
     const resolvedState =
       initialState === undefined
         ? this.initialState
-        : withServiceScope(this, () => {
```

Comment:
I've found it not be required as `machine.resolveState` just resolves `.configuration` and `.value`, it cant call `spawn` in the process (which is the only thing needing the "service scope" right now)